dist/utils.js

```diff
@@ -21,23 +21,52 @@ function transformToValidEnvName(secretName) {
 exports.transformToValidEnvName = transformToValidEnvName;
 function extractAlias(input) {
     const parsedInput = input.split(",");
+    let secretRef = input.trim();
+    let secretPath = "/";
+    let alias = transformToValidEnvName(secretRef);
+    let secretName = secretRef;
     if (parsedInput.length > 1) {
-        const alias = parsedInput[0].trim();
-        const secretName = parsedInput[1].trim();
+        alias = parsedInput[0].trim();
+        secretRef = parsedInput[1].trim();
+        secretName = secretRef;
         const validateEnvName = transformToValidEnvName(alias);
         if (alias !== validateEnvName) {
             throw new Error(`The alias '${alias}' is not a valid environment name. Please verify that it has uppercase letters, numbers, and underscore only.`);
         }
-        return [alias, secretName];
     }
-    return [transformToValidEnvName(input.trim()), input.trim()];
+    if (secretRef.startsWith("/")) {
+        [secretName, secretPath] = splitNameAndPath(secretRef);
+        if (parsedInput.length == 1) {
+            alias = transformToValidEnvName(secretName);
+        }
+    }
+    return [alias, { name: secretName, path: secretPath }];
 }
 exports.extractAlias = extractAlias;
-function getSecretValue(api, secretName) {
+function splitNameAndPath(ref) {
+    const s = ref.split("/");
```

Comment:
Simpler and more efficient to use `lastIndexOf()`:

```ts
let name = "";
let path = "";
const sep = ref.lastIndexOf("/");
if (sep > 0) {
   path = ref.substring(0, sep);
   name = ref.substring(sep + 1);
}
return [name, path];
```

Reply:
Thx